modules/images-other-jenkins-config-kubernetes.adoc

```diff
@@ -9,14 +9,18 @@
 The {product-title} Jenkins image includes the pre-installed link:https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes plug-in] that allows Jenkins agents to be dynamically provisioned on multiple container hosts using Kubernetes and {product-title}.
 
 To use the Kubernetes plug-in, {product-title} provides images that are suitable for use as Jenkins agents, including the Base, Maven, and Node.js images.
+// UPDATE?
 
 Both the Maven and Node.js agent images are automatically configured as Kubernetes pod template images within the {product-title} Jenkins image configuration for the Kubernetes plug-in. That configuration includes labels for each of the images that can be applied to any of your Jenkins jobs under their Restrict where this project can be run setting. If the label is applied, jobs run under an {product-title} pod running the respective agent image.
+// UPDATE?
 
 [IMPORTANT]
 ====
 In {product-title} 4.10 and later, the recommended pattern for running Jenkins agents using the Kubernetes plug-in is to use pod templates with both `jnlp` and `sidecar` containers. The `jnlp` container uses the {product-title} Jenkins Base agent image to facilitate launching a separate pod for your build. The `sidecar` container image has the tools needed to build in a particular language within the separate pod that was launched. Many container images from the Red Hat Container Catalog are referenced in the sample image streams present in the `openshift` namespace. The {product-title} Jenkins image has two pod templates named `java-build` and `nodejs-builder` with sidecar containers that demonstrate this approach. These two pod templates use the latest Java and NodeJS versions provided by the `java` and `nodejs` image streams in the `openshift` namespace.
+// UPDATE?
```

Comment:
this section is relevant and it is up to date.
The other sections telling about nodejs and maven agent image should have a link to this section.